Sandbox/Program.cs

```diff
@@ -199,12 +199,18 @@ public static void PlayOneOnOne(IBot bot1, IBot bot2)
                             while (true)
                             {
                                 var commands1 = bot1.Act(gameState1);
-                                client1.SendCommands(commands1);
+                                if (commands1.Any())
```

Comment:
The hang at 1:00 +- 15s still seems to be happening... this was an attempt to narrow down a bit, as I encountered a timeout when sending an empty command list.

Reply:
I don't think that's likely because that must be happening constantly, although I moved that logic to the client anyway because there's no point in sending such a request. (See meta-pull-request.)